P5/Source/Specs/att.locatable.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<classSpec xmlns="http://www.tei-c.org/ns/1.0" module="tei"
+           type="atts" ident="att.locatable" predeclare="true">
+  <desc versionDate="2021-07-29" xml:lang="en">provides attributes for referencing locations by pointing to entries in a canonical list of places.</desc>
+  <attList>
+    <attDef ident="where" usage="opt">
+      <desc versionDate="2021-07-29" xml:lang="en">indicates one or more locations by pointing to a <gi>place</gi> element</desc>
```

Comment:
If we are developing enough spine to actually say what multiple values on `@where` means, I would suggest something like
~~~xml
<p>This attribute may be used to point to multiple canonical
  descriptions simultaneously, in which case the implication is that
  the reference is to multiple places. For example, the November 2015
  Paris Attacks took place at 6 different locations in Paris; this
  might be encoded as an <gi>event</gi> with a <att>where</att>
  attribute with 6 space-separated values.</p>
~~~
But I bet you (@ebeshero or @lb42) could do better than that.
In the prose in ND we should discuss how to encode “here are several places you can go for more information about the place this event took place or where this conversion was used” and “here are several locations, one of which is where this event took place or this conversion was used”. I do not actually know off the top of my head how to encode “here are several locations, each of which may or may not have been a place this event happened or this conversion was used”.

Reply:
@sydb I believe the example provided by Till Grallert on the ticket provides exactly this use, and the idea is quite simply that ALL of the locations are where this conversion is used. I don't believe any other modification is necessary.

If we aren't sure and are providing guesses, well, that's a different problem that would seem to require a different encoding. Perhaps we develop a spine enough to say, this is simply for **indicating** multiple locations for an event or where a conversion was used, not for creating a list of possibilities. I don't think it should be used in that way at all.

Reply:
@sydb I am frankly exhausted and it is close to release time. 

 I can add prose that explains exactly what multiple values of `@where` should mean, and I believe the place for that is in att.locatable, possibly ND and HD also (where `<conversion>` is exemplified).  But I think it's out of scope of this PR and the release of att.locatable to explain how to encode where you can go for more information (and honestly I don't think that's even necessary, since it sounds like you're talking about pointing to a bibliography anyway.) I don't think it's necessary for me right now before this release to write new examples to that represent *one of a list of possible* locations for a conversion or an event or conversion. The only think I'm aware of is that the encoder might want to use `<cert>` or `@cert`, and perhaps in those cases the encoder would be better off itemizing each location singly to specify the certainty level (whatever that means). Better yet, *only* point to locations in `@where` when you're confident of your assertions. When you want to express a lack of certainty, you could always do so the old-fashioned way with a nice `<note>` in which you discuss and point to those locations and encode your low level of certainty or confidence. 

Can we agree, in the interest of time, that the only _necessary_ thing to do right now before this release is to clarify that multiple values of `@where` simply mean multiple places at which a specific conversion applies, and multiple locations where an event occurred. 

If you want to take time to provide some encoding that shows people how to point to locations they have little confidence are actually accurate, please do so. I can't do that this week. I'm sorry.

Reply:
@sydb I should probably add, I think you could well open a new ticket requesting examples that invite you to point with low confidence to one or some of a set, or to point to a full list of locations where more information is available. If we can agree that this is simply not what att.locatable was designed to do, can we please pause this for now and pick it up later in a new ticket?